test/lib/fixed_length/parser_test.rb

```diff
@@ -0,0 +1,73 @@
+require 'test_helper'
+
+class ParserTest < ActiveSupport::TestCase
+  prepend LoadBankFixture
```

Comment:
Actually, looking at it now, I have no idea why I used it. I could've just included `LoadBankFixture` and called `super` from `ParserTest#setup'. 

I *think* I had an actual valid use case to begin with.